packages/main/src/Menu.ts

```diff
@@ -343,6 +404,16 @@ class Menu extends UI5Element {
 
 	_itemClick(e: CustomEvent<ListItemClickEventDetail>) {
 		const item = e.detail.item as MenuItem;
+		const prevSelected = item.isSelected;
+		const parent = item.parentElement as MenuItem | Menu | MenuItemGroup;
+
+		if (parent && "isGroup" in parent && parent.isGroup) {
```

Comment:
Perhaps we could extract this condition in a method that is more descriptive ?
Something like 

```ts
_isGroupParent(parent: MenuItem | Menu | MenuItemGroup) {
    return parent && "isGroup" in parent && parent.isGroup;
}
```

Reply:
done